feincms_extensions/content_types.py

```diff
@@ -0,0 +1,10 @@
+from feincms.content.richtext.models import RichTextContent
+
+
+class JsonRichTextContent(RichTextContent):
+    class Meta(RichTextContent.Meta):
+        abstract = True
+
+    def json(self, **kwargs):
+        """Return a json serializable dictionary containing the content."""
+        return {'text': self.text}
```

Comment:
Given that this is rich text, `html` might be a more appropriate key than `text`.